docs/features/bgp-integration/route-advertisements.md

```diff
@@ -401,14 +406,14 @@ and will egress the cluster towards the provider network; and if the provider
 network is able to route it back to the cluster by virtue of learned BGP routes,
 the traffic will still be dropped to upkeep the CUDN isolation promise.
 
-In the future, different alternatives will be provided to allow interconnecting
-user defined networks.
+OVN-Kubernetes can be configured in "loose advertised UDN isolation mode" using
+the configuration flag `advertised-udn-isolation-mode` set to `loose`. In this
+configuration, if the provider network routes back to the cluster traffic
+originating from a CUDN addressing a different CUDN, the traffic will be
+allowed.
```

Comment:
This sentence is confusing. How about just saying `cross-CUDN pod-to-pod communication via external routing is allowed in the loose mode`.